src/service/service.py

```diff
@@ -142,7 +149,21 @@ async def message_generator(user_input: StreamInput) -> AsyncGenerator[str, None
     yield "data: [DONE]\n\n"
 
 
-@app.post("/stream")
+def _sse_response_example() -> dict[str, Any]:
+    return {
+        status.HTTP_200_OK: {
+            "description": "Server Sent Event Response",
+            "content": {
+                "text/event-stream": {
+                    "example": "data: {'type': 'token', 'content': 'Hello'}\n\ndata: {'type': 'token', 'content': ' World'}\n\n",
```

Comment:
nit: should we add `data: [DONE]\n\n` here for completeness / correctness? Otherwise this change looks good TY!

Reply:
I think a better approach is to use `event` to mark the DONE event, rather than using a specific string in the data.

At present, you might only need two events:  `data` and  `done`.

If you add the event, it also has a great visual effect in Postman.

https://html.spec.whatwg.org/multipage/server-sent-events.html#server-sent-events-intro

![image](https://github.com/user-attachments/assets/3e47875f-3481-4e04-9105-9ec0bd1da43a)

Reply:
Sorry I’m not an expert in this and not sure I’m following the change you’re suggesting. Is it something to include in this PR or a follow on?

Reply:
![image](https://github.com/user-attachments/assets/965a6b1b-4d4d-4aa4-8e5a-57b993454c43)

Just modify the `yield` part to use the function mentioned above.

Reply:
> Sorry I’m not an expert in this and not sure I’m following the change you’re suggesting. Is it something to include in this PR or a follow on?

I think changes can be made in a future PR.

Reply:
Ok - for now the example response is not an accurate example since it doesn’t include the current [DONE] string - mind adding it? And then fix to whatever in future PR

Reply:
Updated!